js/main.js

```diff
@@ -32,6 +60,42 @@ function entryFormData(event) {
   data.entries.unshift(formInputs);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  var newEntry = entryViewCreation(data.entries[0]);
+  ul.prepend(newEntry);
+  switchView('entries');
 }
 
 $form.addEventListener('submit', entryFormData);
+
+function journalEntryView(event) {
+  switchView(data.view);
+  if (data.entries.length === 0) {
+    var noEntries = document.querySelector('div.justify-center');
+    noEntries.className = 'row justify-center';
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    var displayEntries = entryViewCreation(data.entries[i]);
+    ul.appendChild(displayEntries);
+  }
+}
+
+window.addEventListener('DOMContentLoaded', journalEntryView);
+
+function switchView(string) {
+  for (var i = 0; i < $dataViewDivs.length; i++) {
+    if ($dataViewDivs[i].getAttribute('data-view') !== string) {
+      $dataViewDivs[i].className += ' ' + 'hidden';
+    } else {
+      $dataViewDivs[i].className = 'container';
+    }
+  }
+  data.view = string;
+}
+
+document.addEventListener('click', function (event) {
+  if (event.target.textContent === 'Entries') {
```

Comment:
I would add a class to the element instead of working off of text content. If anybody ever changed the text in this link, the link would stop working.